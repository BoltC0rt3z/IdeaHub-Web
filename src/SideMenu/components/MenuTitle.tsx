import React from "react";
import { Row, Col } from "antd";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import { IconDefinition } from "@fortawesome/free-regular-svg-icons";

interface Props {
  icon: IconDefinition;
  label: string;
}

const MenuTitle: React.FC<Props> = props => (
  <Row type="flex" gutter={16}>
    <Col>
      <FontAwesomeIcon icon={props.icon} style={{ fontSize: 20 }} />
    </Col>
    <Col style={{ fontWeight: "bold" }}>{props.label}</Col>
  </Row>
);

export default MenuTitle;
