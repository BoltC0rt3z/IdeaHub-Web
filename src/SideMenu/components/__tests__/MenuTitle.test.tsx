import React from "react";
import { mount, ReactWrapper } from "enzyme";

import MenuTitle from "../MenuTitle";
import { faClock } from "@fortawesome/free-regular-svg-icons";

describe("Menu Title tests", () => {
  const props = {
    icon: faClock,
    label: "My Spaces"
  };

  const wrapper: ReactWrapper = mount(<MenuTitle {...props} />);

  it("Tests for rendering of menu title without crushing", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
