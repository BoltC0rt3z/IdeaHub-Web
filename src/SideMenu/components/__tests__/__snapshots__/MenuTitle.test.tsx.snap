// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu Title tests Tests for rendering of menu title without crushing 1`] = `ReactWrapper {}`;
